### PR TITLE
backend/iptables: rename vars

### PR DESCRIPTION
**Pull request by dougsland**
Use some friendly name for more compressive understanding
of the code.

 Example, now we can use:
      - service.Name
      - service.Namespace
      - endpointEntry.IPs.V4
      - endpointEntry.IPs.V6

Signed-off-by: Douglas Schilling Landgraf <dlandgra@redhat.com>

### CODE REVIEW

**Comment by dougsland** [comment]
$ make test
go install golang.org/x/lint/golint@latest
./hack/test_unit.sh
==================================
resolving mod files
==================================
downloading mods in ./cmd
downloading mods in ./api
downloading mods in ./backends/ipvs-as-sink
downloading mods in ./backends/iptables
downloading mods in ./backends/nft
downloading mods in ./client
downloading mods in ./server
downloading mods in ./examples/iptables-extip
downloading mods in ./examples/userspace-proxier
downloading mods in ./examples/print-state
downloading mods in ./examples/pipe-exec
==================================
running all tests
==================================
testing ./api/localnetv1
==================================
=== RUN   TestInsertString
--- PASS: TestInsertString (0.00s)
=== RUN   ExampleIPSetAdd
--- PASS: ExampleIPSetAdd (0.00s)
=== RUN   ExampleIPSetDiff
--- PASS: ExampleIPSetDiff (0.00s)
PASS
ok  	sigs.k8s.io/kpng/api/localnetv1	0.004s
==================================
testing ./client
==================================
testing: warning: no tests to run
PASS
ok  	sigs.k8s.io/kpng/client	0.005s
==================================
testing ./client/localsink/fullstate
==================================
=== RUN   TestAddRemoveService
    fullstate_test.go:18: callback received 1 seps: 
    fullstate_test.go:20: -  &{Namespace:"test"  Name:"nginx" []}
    fullstate_test.go:18: callback received 0 seps: 
--- PASS: TestAddRemoveService (0.00s)
=== RUN   TestLess
--- PASS: TestLess (0.00s)
=== RUN   TestEqual
--- PASS: TestEqual (0.00s)
PASS
ok  	sigs.k8s.io/kpng/client/localsink/fullstate	0.004s
==================================
testing ./client/pkg/diffstore
==================================
=== RUN   ExampleDiffStore
--- PASS: ExampleDiffStore (0.00s)
PASS
ok  	sigs.k8s.io/kpng/client/pkg/diffstore	0.002s
==================================
testing ./client/serviceevents
==================================
=== RUN   Example
--- PASS: Example (0.00s)
=== RUN   ExampleWrap
--- PASS: ExampleWrap (0.00s)
PASS
ok  	sigs.k8s.io/kpng/client/serviceevents	0.006s
==================================
testing ./server/pkg/endpoints
==================================
=== RUN   ExampleForNodeWithTopology
--- PASS: ExampleForNodeWithTopology (0.00s)
PASS
ok  	sigs.k8s.io/kpng/server/pkg/endpoints	0.006s
==================================
testing ./server/pkg/proxystore
==================================
=== RUN   TestSessionAffinitySetClientIP
--- PASS: TestSessionAffinitySetClientIP (0.00s)
=== RUN   Example
--- PASS: Example (0.00s)
PASS
ok  	sigs.k8s.io/kpng/server/pkg/proxystore	0.004s

**Comment by dougsland** [comment]
I know var rename is not a good thing but helps a lot understand the code.

**Comment by VivekThrivikraman-est** [comment]
@dougsland Some of these code and files in the current iptable backend will be cleaned up in couple of weeks, as some of the code/logic is stale, but is there because of porting ,but no more required. Also I am doing a poc for iptable which would not apply all rule every time when there is a change. I thought of cleaning up these once that is complete, otherwise it would be 2 time cleanup. I think it would be better to wait for a couple of weeks so that we can work on an almost final code?

**Comment by dougsland** [comment]
> @dougsland Some of these code and files in the current iptable backend will be cleaned up in couple of weeks, as some of
> the code/logic is stale, but is there because of porting ,but no more required. Also I am doing a poc for iptable which would not
> apply all rule every time when there is a change. I thought of cleaning up these once that is complete, otherwise it would be 2
> time cleanup. I think it would be better to wait for a couple of weeks so that we can work on an almost final code?

@VivekThrivikraman-est  IMHO, I would merge this one for code clarity (more people can understand quickly what we have now) and in parallel keep working in the POC/cleanup you have in mind. :) Sounds a good plan?

**Comment by VivekThrivikraman-est** [comment]
> > @dougsland Some of these code and files in the current iptable backend will be cleaned up in couple of weeks, as some of
> > the code/logic is stale, but is there because of porting ,but no more required. Also I am doing a poc for iptable which would not
> > apply all rule every time when there is a change. I thought of cleaning up these once that is complete, otherwise it would be 2
> > time cleanup. I think it would be better to wait for a couple of weeks so that we can work on an almost final code?
> 
> @VivekThrivikraman-est IMHO, I would merge this one for code clarity (more people can understand quickly what we have now) and in parallel keep working in the POC/cleanup you have in mind. :) Sounds a good plan?

Yes, sounds good .

File: backends/iptables/endpoints.go [hunk 1]
```diff
@@ -189,46 +189,49 @@ func (em EndpointsMap) apply(ect *EndpointChangeTracker, staleEndpoints *[]Servi
 
 // Merge ensures that the current EndpointsMap contains all <service, endpoints> pairs from the EndpointsMap passed in.
 func (em EndpointsMap) merge(other EndpointsMap) {
-	for svcPortName, epInfo := range other {
-		for name, ep := range *(epInfo) {
-			if ep == nil {
+	for service, endpoint := range other {
```
Comment (VivekThrivikraman-est) [inline backends/iptables/endpoints.go:192]: Could be name endpoints for better clarity.

File: backends/iptables/endpoints.go [hunk 2]
```diff
@@ -189,46 +189,49 @@ func (em EndpointsMap) apply(ect *EndpointChangeTracker, staleEndpoints *[]Servi
...
 		}
 	}
 }
 
 // GetLocalEndpointIPs returns endpoints IPs if given endpoint is local - local means the endpoint is running in same host as kube-proxy.
 func (em EndpointsMap) getLocalReadyEndpointIPs() map[types.NamespacedName]sets.String {
 	localIPs := make(map[types.NamespacedName]sets.String)
-	for svcPortName, epList := range em {
-		for _, ep := range *epList {
+	for service, endpoint := range em {
```
Comment (VivekThrivikraman-est) [inline backends/iptables/endpoints.go:220]: could be renamed to endpoints.

**Pull request merged**